commit 9c62ad2480c074a76664960cc3490178245af5a6
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-06-13T16:52:24+01:00 (4 months ago)
    
    OS-7822 build_jenkins shouldn't run smartos-release for debug builds

diff --git a/tools/build_jenkins b/tools/build_jenkins
index 0660618e..a8ffcefc 100755
--- a/tools/build_jenkins
+++ b/tools/build_jenkins
@@ -173,12 +173,12 @@ case "${PLATFORM_BUILD_FLAVOR}" in
         ;;
 esac
 
-# Update Manta snaplinks for smartos, only if this is a release build
-case "${PLATFORM_BUILD_FLAVOR}" in
-    'smartos'|'triton-and-smartos')
-        if [[ -n "${ENGBLD_RELEASE}" ]]; then
+# Update Manta snaplinks for smartos, only if this is a non-debug release build
+if [[ -z "${ENGBLD_CONFIGURE_DEBUG_ARG}" && -n "${ENGBLD_RELEASE}" ]]; then
+    case "${PLATFORM_BUILD_FLAVOR}" in
+        'smartos'|'triton-and-smartos')
             log_cmd env TRACE=1 ./tools/smartos-release \
                 "${BRANCH}" "${TIMESTAMP}"
-        fi
-        ;;
-esac
+            ;;
+    esac
+fi
